Makefile

```diff
@@ -19,7 +19,11 @@ get:
 
 .PHONY: test
 test: get
-	go test -v -short $(TOPLEVEL_DIRS)
+	go test -v -short -tags unit $(TOPLEVEL_DIRS)
```

Comment:
Why do we need both `-short` and `-tags`? Would be better to do such things in separate PRs.

Reply:
agree with @gtopper. I think we should get rid of the short and use tags for test suites